lib/main.dart

```diff
@@ -11,115 +12,11 @@ class MyApp extends StatelessWidget {
-
-  void _incrementCounter() {
-    setState(() {
-      // This call to setState tells the Flutter framework that something has
-      // changed in this State, which causes it to rerun the build method below
-      // so that the display can reflect the updated values. If we changed
-      // _counter without calling setState(), then the build method would not be
-      // called again, and so nothing would appear to happen.
-      _counter++;
-    });
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    // This method is rerun every time setState is called, for instance as done
-    // by the _incrementCounter method above.
-    //
-    // The Flutter framework has been optimized to make rerunning build methods
-    // fast, so that you can just rebuild anything that needs updating rather
-    // than having to individually change instances of widgets.
-    return Scaffold(
-      appBar: AppBar(
-        // TRY THIS: Try changing the color here to a specific color (to
-        // Colors.amber, perhaps?) and trigger a hot reload to see the AppBar
-        // change color while the other colors stay the same.
-        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
-        // Here we take the value from the MyHomePage object that was created by
-        // the App.build method, and use it to set our appbar title.
-        title: Text(widget.title),
-      ),
-      body: Center(
-        // Center is a layout widget. It takes a single child and positions it
-        // in the middle of the parent.
-        child: Column(
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          //
-          // TRY THIS: Invoke "debug painting" (choose the "Toggle Debug Paint"
-          // action in the IDE, or press "p" in the console), to see the
-          // wireframe for each widget.
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headlineMedium,
-            ),
-          ],
-        ),
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: const Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
+      home: const Onboarding(),
```

Comment:
something to consider is that the Onboaring screen will only be presented on first launch. For example: it will only appear if the user opens the app for the first time. After that the home property will be a login screen. What do you think?

Reply:
@luismir15 I have taken your suggestion into consideration and made the necessary updates in the PR. You can re-review now.

Reply:
@akarsh-jain-790 awesome it's working! But I deleted the app to see if it works. I ran the app again, but it still opens up in the sign in view, are you getting the same behavior?

Reply:
@luismir15 I just found the reason behind the behavior you mentioned. It seems like the issue is related to the Flutter framework initialization. When the app is launched for the first time, the framework initializes properly, and the onboarding screen is shown. However, when you delete and reinstall the app, the framework initialization is not triggered, resulting in the app opening directly to the sign-in view.

To fix this, we need to ensure that the Flutter framework is initialized properly on app launch. You can do this by calling `WidgetsFlutterBinding.ensureInitialized()` in the main function of your app. This will ensure that the binding is initialized and the proper flow is maintained.

Give it a try, and let me know if you still face any issues.

Reply:
Beautiful! let's go ahead with the merge